src/com/dotmarketing/viewtools/content/BinaryMap.java

```diff
@@ -159,5 +159,24 @@ public String getThumbnailUri(Integer width, Integer height, String background){
 	public File getFile() {
 		return file;
 	}
-	
+	public int getHeight() {
+		int height = 0;
+		try {
+			height = ImageUtil.getInstance().getDimension(getFile()).height;
```

Comment:
Rogelio's above suggestion applies here as well:

> I think we could modify the code a little to call one time ImageUtil.getInstance().getDimension(getFileAsset()) and store the dimension in a private variable, so that way you don't load the file twice to get height or width in the same instance.
Just check if dimension is not null and get the proper value